Swat/SwatHtmlHeadEntrySet.php

```diff
@@ -90,62 +81,47 @@ public function addEntry($entry)
         }
     }
 
-    // }}}
-    // {{{ public function addEntrySet()
-
     /**
-     * Adds a set of HTML head entries to this set
+     * Adds a set of HTML head entries to this set.
      *
-     * @param SwatHtmlHeadEntrySet $set the set to add.
+     * @param SwatHtmlHeadEntrySet $set the set to add
      */
     public function addEntrySet(SwatHtmlHeadEntrySet $set)
     {
         $this->entries = array_merge($this->entries, $set->entries);
     }
 
-    // }}}
-    // {{{ public function toArray()
-
     public function toArray()
     {
         return $this->entries;
     }
 
-    // }}}
-    // {{{ public function count()
-
     /**
-     * Gets the number of entries in this set
+     * Gets the number of entries in this set.
      *
      * Fulfills the Coutnable interface.
      *
-     * @return integer the number of entries in this set.
+     * @return int the number of entries in this set
      */
-    public function count()
+    public function count(): int
     {
         return count($this->entries);
     }
 
-    // }}}
-    // {{{ public function getIterator()
-
     /**
-     * Gets an iterator over the entries in this set
+     * Gets an iterator over the entries in this set.
      *
      * Fulfills the IteratorAggregate interface.
      *
-     * @return Iterable an iterator over the entries in this set.
+     * @return iterable an iterator over the entries in this set
      */
-    public function getIterator()
+    public function getIterator(): Traversable
```

Comment:
can you use `iterable` and maintain the existing implementation?

Reply:
The `IteratorAggregate` interface has `getIterator(): Traversable`, so ... no?